cypress/integration/Unit/details.spec.js

```diff
@@ -0,0 +1,24 @@
+/// <reference types="cypress" />
+import jwt from 'jsonwebtoken';
+describe('Documents', () => {
+  it('Has a link', () => {
+    const token = jwt.sign(
+      { groups: ['housingneeds-singleview-beta'] },
+      'a-secure-signature'
+    );
+    cy.setCookie('hackneyToken', token, {
+      url: 'http://localhost:3001',
+      domain: 'localhost'
+    });
+    cy.visit('http://localhost:3001/customers/5/view');
+
+    cy.get('tbody > tr > td > a').each($el =>
```

Comment:
can we specify that we want to get the links from the table within the `.activity` div